test/e2e/.golangci.yml

```
@@ -0,0 +1,67 @@
+      - default
+      - prefix(k8s.io,sigs.k8s.io)
+      - prefix(github.com/ovn-org)
+      - localmodule
+      - dot
+
+  govet:
+    enable-all: true
+    disable:
+      - fieldalignment
+      - shadow
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)